Find a functionally equivalent code:com.android.layoutlib.bridge.impl.RenderSessionImpl.moveChild:COMMENT
Method Modifier: public      
Comment:/**
 * Moves a view to a new parent at a given location
 * <p>
 * {@link #acquire(long)} must have been called before this.
 *
 * @throws IllegalStateException if the current context is different than the one owned by
 * the scene, or if {@link #acquire(long)} was not called.
 *
 * @see RenderSession#moveChild(Object, Object, int, Map, IAnimationListener)
 */

@@ -17,28 +17,28 @@
                 @Override
                 public void run() {
                     Result result = moveView(previousParent, newParentView, childView, index, params);
-                    if (result.isSuccess() == false) {
+                    if (!result.isSuccess()) {
                         listener.done(result);
                     }
                     // ready to do the work, acquire the scene.
                     result = acquire(250);
-                    if (result.isSuccess() == false) {
+                    if (!result.isSuccess()) {
                         listener.done(result);
                         return;
                     }
@@ -71,77 +71,77 @@
         return SUCCESS.createResult(layoutParams);
     }
     Result result = moveView(previousParent, newParentView, childView, index, layoutParams);
-    if (result.isSuccess() == false) {
+    if (!result.isSuccess()) {
         return result;
     }
     result = render(false);

